tests/test_messages.py

```diff
@@ -115,6 +116,27 @@ def test_request_paths():
     )
 
     assert message == Message.deserialize(message).serialize_data()
+    message['message_type'] = 'PathsRequest'
+    assert isinstance(Message.deserialize(message), PathsRequest)
+
+
+def test_paths_reply():
+    message: Dict = dict(
+        message_type='PathsReply',
+        token_network_address='0x82dd0e0eA3E84D00Cc119c46Ee22060939E5D1FC',
+        target_address='0x82dd0e0eA3E84D00Cc119c46Ee22060939E5D1FC',
+        value=1000,
+        num_paths=1,
+        chain_id=1,
+        nonce=1,
+        paths=['0x82dd0e0eA3E84D00Cc119c46Ee220609', '0xA3E84D00Cc119c46Ee22060939E5D1FC'],
+        estimated_fees=[10000],
```

Comment:
this looks strange, the list of paths is just one path, you should make sure that it's properly defined.

Reply:
Done!